internal/flags/flags.go

```diff
@@ -96,10 +97,12 @@ type Flags struct {
 	SambaService                        smb.ServiceInterface
 	MEBxPassword                        string
 	ConfigTLSInfo                       ConfigTLSInfo
+	pr                                  utils.PasswordReader
```

Comment:
no other variable in this struct follows this naming convention. For the sake of consistency please use `passwordReader`

Reply:
done